src/renderer/helpers/assetHelper.test.ts

```diff
@@ -358,14 +342,6 @@ describe('helpers/assetHelper', () => {
   })
 
   describe('getAssetFromNullableString', () => {
```

Comment:
don't delete, replace the test with another asset

Reply:
Done